package.json

```diff
@@ -12,11 +12,12 @@
   "private": true,
   "scripts": {
     "lint": "eslint --quiet \"**/*.js\"",
+    "stylelint": "stylelint --quiet --allow-empty-input \"./packages/stylelint-plugin-ckeditor5-rules/style.css\" --fix",
```

Comment:
What is this?

Reply:
Used for testing, commited by mistake, sorry.